setup.cfg

```diff
@@ -5,7 +5,7 @@ desktop_files=[('share/applications', 'po/uh/', ('content/packages/unknown-horiz
 [pycodestyle]
 exclude=horizons/ext/, horizons/gui/translations.py, horizons/constants.py
 count=1
-select = W2, W3, W6, E11, E125, E13, E20, E21, E22, E23, E242, E25, E27, E301, E302, E303, E305, E70, E713
+select = W2, W3, W6, E11, E125, E13, E20, E21, E22, E23, E242, E25, E27, E3, E70, E713
```

Comment:
Doesn't the commit message say, it's only for E301? This looks like an error, as it removes more and only leaves `E3`, which wasn't here before.

Reply:
E3 summarizes all E3xx checks, in this case E301 to E306.

Reply:
Ah ok. Thank you for the explanation. This means, E304 is now included, too, but wasn't before.